public/pages/DetectorsList/utils/tableUtils.tsx

```diff
@@ -59,15 +134,15 @@ export const staticColumn = [
     ),
     sortable: true,
     dataType: 'number',
-    align: 'center',
-    width: '100px',
+    align: 'right',
+    truncateText: false,
   },
   {
     field: 'lastActiveAnomaly',
     name: (
-      <EuiToolTip content="Last active anomaly time">
+      <EuiToolTip content="Time of the last active anomaly with a grade > 0">
```

Comment:
have you confirmed the wording? It is `Last anomaly occurrence` as per [mockup](https://aws-uxdr.invisionapp.com/d/?redirHash=#/console/19392952/405140830/comments?scrollOffset=4200)

Reply:
nvm. This is tooltip